spinalcordtoolbox/scripts/sct_create_mask.py

```diff
@@ -293,7 +295,11 @@ def create_line(param, fname, coord, nz):
     copy(fname, 'line.nii', verbose=param.verbose)
 
     # set all voxels to zero
-    run_proc(['sct_maths', '-i', 'line.nii', '-mul', '0', '-o', 'line.nii'], param.verbose)
+    img = Image('line.nii')
+    data = get_data_or_scalar('0', img.data)
+    data_concat = concatenate_along_4th_dimension(img.data, data)
+    img.data = np.prod(data_concat, axis=3)
```

Comment:
Would it be better to merge these 3 lines into a function called, say, `multiply()` in `math.py`? Since we're repeating ourselves here and also in:

https://github.com/neuropoly/spinalcordtoolbox/blob/6129ec7ccb16b777462d57276e1f63498a1f9ef2/spinalcordtoolbox/scripts/sct_maths.py#L318-L321

(just to make things less error prone if these lines ever have to change in the future)

Reply:
https://github.com/neuropoly/spinalcordtoolbox/issues/3189